syntax = "proto2";
package spider;

message consume_url_req {
	repeated string urls = 1;
}
